src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js

```diff
@@ -9696,81 +9696,83 @@ oop.inherits(Mode, CMode);
 exports.Mode = Mode;
```

Comment:
There shouldn't be any changes to ace-uncompressed.js as part of this PR (it's a generated file).